### PR TITLE
route response values

### PR DESCRIPTION
**Pull request by ericwenn**
- feat(routingv8): add notice to route response
- fix(routingv8)!: remove error codes from routes response
- feat(routingv8): add polyline on route sections

### CODE REVIEW

**Review by thall — APPROVED** [review]

File: routingv8/response.go [hunk 1]
```diff
@@ -69,8 +71,44 @@ type Route struct {
...
+	// for possible values.
+	Code string `json:"code"`
+	// Describes the impact a notice has on the resource to which the notice is attached.
+	Severity NoticeSeverity `json:"severity"`
+}
+
+type NoticeSeverity string
+
+const (
+	// CriticalNoticeSeverity is used to indicate that the notice is must not be ignored,
```
Comment (philiphassel) [inline routingv8/response.go:101]: nit: "is must"

**Review by philiphassel — APPROVED** [review]
Looks good to me ⭐

**Comment by KrisRosDim** [comment]
Just an FYI, Polylines tend to be considerably weighty, i.e. hundreds of thousands of points

**Comment by ericwenn** [comment]
> Just an FYI, Polylines tend to be considerably weighty, i.e. hundreds of thousands of points

Yep, we might chose not to use it (ie not specify we want them in request), but having them in the response object should be fine (they're zero values if not set).

**Pull request merged**